certified-connectors/Converter by Power2Apps/apiDefinition.swagger.json

```diff
@@ -3,676 +3,2194 @@
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/FileData"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/FileResponse"
+            }
+          }
+        },
+        "x-ms-visibility": "important",
+        "summary": "Convert Image",
+        "description": "Convert an image to JPEG, JPG, PNG, BMP, GIF, TIFF, .."
+      }
+    },
+    "/GetFileExtensions": {
+      "get": {
+        "tags": [
+          "Converter"
+        ],
+        "operationId": "GetFileExtensions",
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/ListOption"
+              }
+            }
+          }
+        },
+        "x-ms-visibility": "internal",
+        "summary": "Get list of file extensions",
+        "description": "Returns supported file extensions"
+      }
+    },
+    "/ConvertFileToPdf": {
+      "post": {
+        "tags": [
+          "Converter"
+        ],
+        "operationId": "ConvertFileToPdf",
+        "consumes": [
+          "application/json",
+          "text/json",
+          "application/*+json"
+        ],
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "in": "header",
+            "name": "fileName",
+            "type": "string",
+            "x-ms-visibility": "important",
+            "x-ms-summary": "File Name",
+            "description": "File name"
+          },
+          {
+            "in": "header",
+            "name": "fileExtension",
+            "type": "string",
+            "default": "Extract from File name",
+            "x-ms-dynamic-values": {
+              "operationId": "GetFileExtensions",
+              "value-path": "name",
+              "value-title": "name",
+              "parameters": {}
+            },
+            "x-ms-visibility": "important",
+            "x-ms-summary": "File Extension",
+            "description": "File extension"
+          },
+          {
+            "in": "body",
+            "name": "fileData",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/FileData"
+            },
+            "x-ms-visibility": "important",
+            "x-ms-summary": "File",
+            "description": "File"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/FileContentResponse"
+            }
+          }
+        },
+        "x-ms-visibility": "internal",
```

Comment:
@stsiek The operation has been hidden. This change breaks the designer for existing flows.

Reply:
we would like to set this action to deprecated, so it is set to "internal". we had understood that it can still be used for existing flows but is no longer visible when adding new actions. We also tested it and it works fine.